tests/fixtures/groups_none_json

```diff
@@ -1 +1 @@
-[]
+"id":"global-acl-group-id","name":"globalACLGroup","email":"email","status":"Active"
```

Comment:
note: i am loosely familiar with this code; however, this file says `groups_non_json` but you seem to have added a group.  Perhaps there is a different figure file that should be used instead?

Same above

Reply:
@jagadeesh545 Yes; Could you please clarify why the groups fixtures and tests have been modified?

Reply:
There seems to be a default `globalACLGroup` as part of the local vinyldns docker instance that gets launched as part of testing. Hence, I had to update the fixture to account for this default group.

Reply:
Ah, thanks for clarifying and apologies for the confusion. This is likely a side effect of updating to VinylDNS 0.9.7. Perhaps @pauljamescleary can offer some insight?